frontend/packages/console-plugin-sdk/src/utils/extension-i18n.ts

```diff
@@ -0,0 +1,15 @@
+import { TFunction } from 'i18next';
+import { deepForOwn } from '@console/dynamic-plugin-sdk/src/utils/object';
+import { Extension } from '../typings';
+
+export const isTranslatableString = (value): value is string => {
+  return typeof value === 'string' && /^%.+%$/.test(value);
```

Comment:
I don't like micro optimizations, but I also think its not necessary to create/compile/match a regex pattern here.

```suggestion
  return typeof value === 'string' && value.startsWith('%') && value.endsWith('%');
```

Reply:
Agreed, we can use `startsWith '%' and endsWith '%' and length > 2` condition instead of RegExp here.